S4P-65 Manager - Pet CRUD

SonNVQ opened the pull request:
-/manager/pet
-/manager/pet?page=#
-/manager/pet/add
-/manager/pet/edit?id=?
-POST /manager/pet/delete (id = #)

SonNVQ committed:
S4P-65 Homepage with pagination

SonNVQ committed:
S4P-64 First done of Relief Group CRUD

- /manager/relief-group
- /manager/relief-group/add
- /manager/relief-group/edit?id=#
- POST /manager/relief-group/delete (id = #)

SonNVQ committed:
Merge pull request #15 from SonNVQ/S4P-64-BE-Manager-Relief-Group

S4P-65 Merge Relief Group to Pet

SonNVQ committed:
Merge branch 'S4P-65-BE-Manager-Pet' of https://github.com/SonNVQ/saigon4paws-be into S4P-65-BE-Manager-Pet

SonNVQ committed:
S4P-65 First done of Manager - Pet CRUD

-/manager/pet
-/manager/pet?page=#
-/manager/pet/add
-/manager/pet/edit?id=?
-POST /manager/pet/delete (id = #)

SonNVQ committed:
S4P-65 Fix upload image

xquynh1 reviewed (commented):
- Upload ảnh rồi bấm lưu hiển thị lỗi Whitelabel error
- Chưa validate ô Số tuổi của pet (điền chữ vẫn chấp nhận)

xquynh1 committed:
S4P-65 Increase max upload file size

Tuyet-Trinh commented:
> 

Số tuổi thì tuỳ tình trạng tại mình cũng chưa thống nhất đơn vị tuổi tính tháng hay năm á, nên nhập 1 tháng hay 1 tuổi, hay chưa xác định j đó cũng đc nhỉ🧐

jng151023 committed:
S4P-65 Fix lost image when update pet info without new image

SonNVQ committed:
S4P-65 Refactor upload image function in PetController

-Create FileService:
+String saveFile(String savingPath, String fileName, byte[] fileContent) throws Exception;
+String saveImageFromMultipartFile(String savingFolder, MultipartFile image) throws Exception;

-Move upload code base from PetController to PetService.
-Add image extensions validation.
